src/views/Islands.tsx

```diff
@@ -0,0 +1,19 @@
+import { Game } from '../Game'
+import { Island } from '../Island'
+import { IslandListItem } from '../fragments/IslandListItem'
+import { h } from '../ui'
+
+type Props = { game: Game }
+
+export const Islands = ({ game }: Props) => {
+  const islands: Array<Island> = Island.getIslands(game.name)
+  return (
+    <div>
+      {game.name} empire dominion <span style={{ color: 'gray' }}>{islands.length} islands</span>
+      <hr/>
+      <div>
+        {islands.map(island => { return IslandListItem({ game, island }) })}
```

Comment:
A parte de que te ahorras tokens solo usando la flecha, like:
```suggestion
        {islands.map(island => IslandListItem({ game, island }))}
```

Lo que quiero es que uses expresiones JSX:
```suggestion
        {islands.map(island => <IslandListItem game={game} island={island} />)}
```

Reply:
Añadida en el último commit.